tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileMain.scala

```diff
@@ -154,12 +158,29 @@ object ProfileMain extends Logging {
       healthCheck.listRemovedExecutors()
       healthCheck.listPossibleUnsupportedSQLPlan()
 
+      if (appArgs.generateDot()) {
+        if (appArgs.compare()) {
+          logWarning("Dot graph does not compare apps")
+        }
+        apps.foreach { app =>
+          val start = System.nanoTime()
+          GenerateDot(app, outputDirectory)
+          val duration = TimeUnit.SECONDS.convert(System.nanoTime() - start, TimeUnit.NANOSECONDS)
+          textFileWriter.write(s"Generated DOT graphs for app ${app.appId} " +
+              s"to $outputDirectory in $duration second(s)\n")
+        }
+      }
+
       if (appArgs.generateTimeline()) {
         if (appArgs.compare()) {
           logWarning("Timeline graph does not compare apps")
```

Comment:
same here,

Reply:
Same response here. I want to get the graphs out without having to run the profiling tool twice.